edx_rest_framework_extensions/auth/jwt/middleware.py

```diff
@@ -251,9 +267,43 @@ def process_view(self, request, view_func, view_args, view_kwargs):  # pylint: d
         else:
             attribute_value = 'missing-both'
             log.warning('Both JWT auth cookies missing. JWT auth cookies will not be reconstituted.')
```

Comment:
@feanil: I noticed this in the logs recently. There are a bunch of `log.warning`s here that should be login.info at most, and probably even log.debug. What do you think?

Reply:
That makes sense, it sounds like that would be a follow-up ticket in this repo? Is making an issue for this repo right now, the correct next step?

Reply:
Ticket sounds right.

Reply:
Created https://github.com/openedx/edx-drf-extensions/issues/363